src/conn/conn_tiered.c

```diff
@@ -362,3 +355,23 @@ __wt_tiered_storage_destroy(WT_SESSION_IMPL *session)
 
     return (ret);
 }
+
+/*
+ * __wt_tiered_storage_init --
+ *     Configure tiered storage initially.
+ */
+int
+__wt_tiered_storage_init(WT_SESSION_IMPL *session, const char *cfg[])
+{
+    WT_CONFIG_ITEM auth, bucket, name;
+    WT_CONNECTION_IMPL *conn;
+
+    conn = S2C(session);
+
+    WT_RET(__wt_config_gets(session, cfg, "tiered_storage.name", &name));
+    WT_RET(__wt_config_gets(session, cfg, "tiered_storage.bucket", &bucket));
+    WT_RET(__wt_config_gets(session, cfg, "tiered_storage.auth_token", &auth));
```

Comment:
Alphabetize all these. (And I'll apologize because I see my original code did not!)